src/zNA/index.ts

```diff
@@ -0,0 +1,43 @@
+import { ethers } from 'ethers';
+
```

Comment:
Once the contracts are updated, can you rework this folder to just be `contracts` and use `typechain` to export functionality. See the `zns-sdk` project for an example